# Generated by Django 5.2.3 on 2025-07-30 14:15

from django.db import migrations, models


class Migration(migrations.Migration):

    initial = True

    dependencies = [
    ]

    operations = [
        migrations.CreateModel(
            name='PaymentNotification',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('pay_status', models.IntegerField(choices=[(0, 'Processing'), (1, 'Payment Successful'), (2, 'Payment Failed')])),
                ('pay_time', models.DateTimeField(blank=True, null=True)),
                ('out_trade_no', models.CharField(max_length=255, unique=True)),
                ('transaction_id', models.CharField(max_length=255)),
                ('amount', models.DecimalField(decimal_places=2, max_digits=12)),
                ('actual_payment_amount', models.DecimalField(decimal_places=2, max_digits=12)),
                ('actual_collect_amount', models.DecimalField(decimal_places=2, max_digits=12)),
                ('payer_charge', models.DecimalField(decimal_places=2, max_digits=12)),
                ('payee_charge', models.DecimalField(decimal_places=2, max_digits=12)),
                ('pay_message', models.TextField(blank=True, null=True)),
                ('sign', models.CharField(max_length=64)),
                ('received_at', models.DateTimeField(auto_now_add=True)),
                ('processed', models.BooleanField(default=False)),
                ('processed_at', models.DateTimeField(blank=True, null=True)),
                ('error_message', models.TextField(blank=True, null=True)),
            ],
        ),
    ]
